app/views/info/_vendor_terms_text.html.erb

```diff
@@ -122,17 +129,17 @@
 
 <p>We may ask for information to help verify your identity including a driver’s license or other government issued identification, or a business license. We may request your permission to do a physical inspection at your place of business and to examine bicycles that pertain to your compliance with this Agreement. Your failure to comply with any of these requests within five (5) days may result in deactivation or termination of your Bike Index account. You authorize us to retrieve additional information about you from third parties and other identification services.</p>
 
-<p>After we have collected and verified all your information, Bike Index will review your account and determine if you are eligible to use the Service. Bike Index will notify you once your account has been either approved or deemed ineligible for use of the Service.</p>
+<p>After we have collected and verified all your information, Bike Index will review your Account and determine if you are eligible to use the Service. Bike Index will notify you once your account has been either approved or deemed ineligible for use of the Service.</p>
 
 <p>By accepting the terms of this Agreement, you are providing us with authorization to retrieve information about you by using third parties, including bike industry wholesalers, distributors, and other information providers. You acknowledge that such information retrieved may include your name, address history, business history, and other data about you. Bike Index may periodically update this information to determine whether you continue to meet our eligibility requirements.</p>
 
-<p>You agree that Bike Index is permitted to contact and share information about you and your application (including whether you are approved or declined), and your use of Bike Index with our friends.</p>
+<p>You agree that Bike Index is permitted to contact and share information about you and your application (including whether you are approved or declined), as well as your use of Bike Index with our friends.</p>
```

Comment:
> with our friends

May want to consider elaborating on who our friends are. If I had never heard about Bike Index before and read this before using it, this would raise some questions.